tests/queries/0_stateless/01881_total_bytes_storage_buffer.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS test_buffer_table;
+
+CREATE TABLE test_buffer_table
+(
+    `a` Int64
+)
+ENGINE = Buffer('', '', 1, 1000000, 1000000, 1000000, 1000000, 1000000, 1000000);
+
+SELECT total_bytes FROM system.tables WHERE name = 'test_buffer_table';
```

Comment:
To pass [flaky check](https://clickhouse-test-reports.s3.yandex.net/24450/6bc0167970a4cbf8bef58ead5e325fae89a325e9/functional_stateless_tests_flaky_check_(address).html#fail1), see [style check](https://clickhouse-test-reports.s3.yandex.net/24450/6bc0167970a4cbf8bef58ead5e325fae89a325e9/style_check.html#fail1), the query should contain filter by database, i.e. `and database = currentDatabase()`

Reply:
Thanks. Fixed